src/PowerShellEditorServices/Server/PsesServiceCollection.cs

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation.Host;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.PowerShell.EditorServices.Hosting;
+using Microsoft.PowerShell.EditorServices.Services;
+
+namespace Microsoft.PowerShell.EditorServices.Server
+{
+    internal static class PsesServiceCollection
```

Comment:
Given that this is a static extensions class, maybe call it something like `PsesServiceCollectionExtensions` instead (i.e. it doesn't actually implement an collection of services)